.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
With this file, is easier to debug any method call from vscode, you just need to configure the `args` attribute with the command and argument you want to debug.